client/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fab} from '@fortawesome/free-brands-svg-icons';
+import {faHeart, faCompass, faSearch, faUser} from '@fortawesome/free-solid-svg-icons';
```

Comment:
what the diff between fontawesome and fortawesome?

Reply:
fortawesome is a name of a library for fontawesome icons